aggregators/mtg_cards/cards.js

```diff
@@ -0,0 +1,38 @@
+function MtgCardLoader(){
+    var request = require("request");
```

Comment:
`require` should happen on initialisation of the file, not inside a class constructor.